proto/v1/subscription_service.proto

```diff
@@ -77,3 +83,15 @@ message Subscription {
 
   string org_name = 8 [(google.api.field_behavior) = OUTPUT_ONLY];
 }
+
+message FeatureMatrix {
+  repeated Feature features = 1;
+}
+
+message Feature {
+  // Name is the feature name.
+  string name = 1;
+
+  // Matrix is the feature matrix for different plan. The key is the plan enum in string value.
+  map<string, bool> matrix = 2;
```

Comment:
Consider to use map<Plan, bool> in the future, but not right now. Enum has types.